palm/cli.py

```diff
@@ -95,7 +95,7 @@ def required_dependencies_ready():
             "Docker is not installed, please install it first",
         ),
         (
-            "docker-compose --version",
+            "which docker-compose || where docker-compose",
```

Comment:
@cgaunt-palmetto @omoorepalmetto - I think this should work for windows, if either of you could test this branch I'd be very grateful. Happy to pair up on how to do that!